### DISCUSSION TITLE
Amazon Redshift

### DISCUSSION THREAD

**Discussion by wcauchois** [amacneil/dbmate#198] [Support] [answered]
Does this library support Amazon Redshift? It says it supports postgres which is wire-compatible so I imagine you could maybe use it, but has anyone tried it with Redshift and/or are there any plans to officially support Redshift?

Thanks!

**Comment by amacneil** [comment]
I have not tried it, but if redshift is wire compatible then you should definitely try it out - I would be interested to hear back. According to a [brief web search](https://www.reddit.com/r/golang/comments/61rfgd/connect_to_amazon_redshift_databases/dfgr8xc/) it seems the `lib/pq` library we use should work.

**Comment by dossy** [comment, marked as answer]
Related: #361

**Reply by dossy** [reply]
Redshift support added in [v2.11.0](https://github.com/amacneil/dbmate/releases/tag/v2.11.0) via PR #488.
